plugins/modules/elasticsearch_info.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/python
+# Copyright (c) 2021 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: elasticsearch_info
+version_added: 1.6.0
```

Comment:
```suggestion
version_added: 2.0.0
```

It's currently expected that 2.0.0 will be the next release